packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
-            - `links: object[]`: An array of `MenuLink` or `MenuGroup` objects that are should be grouped together in the group.
+            ## Plugins
 
-            ###### Menus example:
-            ```yaml
-            lowdefy: LOWDEFY_VERSION
-            menus:
-              - id: default
-                links:
-                  - id: page1
-                    type: MenuLink
-                    pageId: page1
-                    properties:
-                      title: Page 1
-                      icon: AiOutlineFile
-                  - id: top-group
-                    properties:
-                      title: Group
-                      icon: AiOutlineGroup
-                    links:
-                      - id: page2
-                        type: MenuLink
-                        pageId: page2 # pageId will be used as link title
-                      - id: external
-                        type: MenuLink
-                        url: https://external.com
-                        properties:
-                          title: External site
-                      - id: nested-group
-                        type: MenuGroup
-                        properties:
-                          title: Nested Group
-                        links:
-                          - id: page3
-                            type: MenuLink
-                            pageId: page3
-                            properties:
-                              title: Page 3
-              - id: page-1-and-3
-                links:
-                  - id: page1
-                    type: MenuLink
-                    pageId: page1
-                    properties:
-                      title: Page 1
-                      icon: AiOutlineFile
-                  - id: page3
-                    type: MenuLink
-                    pageId: page3
-                    properties:
-                      title: Page 3
-                      icon: AiOutlineControl
-            ```
+            [TODO]: <> (# TODO: Add plugins things here)
+
+            ## Menus
+
+            Menu objects are objects with links to pages. They are filtered to only show pages that the user is authorized to see and are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+
+            See more about how menu objects are used and defined [here](/menus).
 
-            # Pages
+            ## Pages
 
             Pages in a Lowdefy app are actually just blocks, the building blocks of a Lowdefy app, with a few extra restrictions and features.
 
             Each page should have an `id` that is unique among all pages in the app. Each page is served with the `pageId` as the url route. That is, if you create a page with id `page1`, it will be served at `domain.com/page1`.
```

Comment:
Also here:
That is, if a page is created with id `page1`, it will be served at ...

Reply:
Change has been made in #1314.